### PR TITLE
Add the board configuration for Number8

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

**Comment by nomis52** [comment]
Blocked behind #146

**Comment by nomis52** [comment]
Synced.

File: firmware/ja-rule.X/nbproject/configurations.xml [hunk 1]
```diff
@@ -237,6 +266,7 @@
         <linkerTool>
           <linkerLibItems>
             <linkerLibFileItem>/opt/microchip/harmony/v1_02/bin/framework/peripheral/PIC32MX795F512L_peripherals.a</linkerLibFileItem>
+            <linkerLibFileItem>../../../../../../opt/microchip/harmony/v1_02/bin/framework/peripheral/PIC32MX795F512L_peripherals.a</linkerLibFileItem>
```
Comment (peternewman) [inline firmware/ja-rule.X/nbproject/configurations.xml]: Any reason this is relative and the one above is absolute?

**Comment by peternewman** [comment]
Generally LGTM, I think.

File: firmware/ja-rule.X/nbproject/configurations.xml (hunk 1 above)
Comment (nomis52) [inline firmware/ja-rule.X/nbproject/configurations.xml]: Nope, good catch. Fixed.

**Comment by peternewman** [comment]
LGTM

**Pull request merged**